src/Plugin_Command.php

```diff
@@ -659,8 +659,8 @@ protected function get_item_list() {
 				'name'           => $name,
 				'status'         => $this->get_status( $file ),
 				'update'         => (bool) $update_info,
-				'update_version' => $update_info['new_version'],
-				'update_package' => $update_info['package'],
+				'update_version' => is_array( $update_info ) ? $update_info['new_version'] : null,
```

Comment:
As we're dealing with WordPress, we should take the safest route possible to make it as robust as possible.

Therefore, it's preferable to use `isset( $update_info['new_version'] )` instead of `is_array( $update_info )`, as it will also cover the case where `$update_info` is an array but does not contain the key `new_version`.

All checks should be updated to that effect.

Reply:
I assume you'd want the call to `isset` to be chained after a call to `is_array`? Otherwise, wouldn't this still trigger the warning?

Reply:
^disregard